src/safeds/data/tabular/transformation/_sequential_table_transformer.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from safeds._utils import _structural_hash
+from safeds.exceptions import TransformerNotFittedError, TransformerNotInvertibleError
+
+from ._invertible_table_transformer import InvertibleTableTransformer
+
+if TYPE_CHECKING:
+    from safeds.data.tabular.containers import Table
+
+    from ._table_transformer import TableTransformer
+
+
+class SequentialTableTransformer(InvertibleTableTransformer):
+    """
+    The SequentialTableTransforrmer transforms a table using multiple transformers in sequence.
+
+    Parameters
+    ----------
+    transformers:
+        The list of transformers used to transform the table. Used in the order as they are supplied in the list.
+
+    Raises
+    ------
+    ValueError:
+        Raises a ValueError if the list of Transformers is None or contains no transformers.
+    """
+
+    def __init__(
+        self,
+        transformers: list[TableTransformer],
+        *,
+        column_names: str | list[str] | None = None,  # noqa: ARG002
+    ) -> None:
+        super().__init__(None)
+
+        # Check if transformers actually contains any transformers.
+        if transformers is None or len(transformers) == 0:
+            raise ValueError("transformers must contain at least 1 transformer")
```

Comment:
The behavior of a sequential transformer that wraps an empty list is well defined, it's simply a no-op. We should, thus, get rid of the error and only issue a warning instead. See e.g. https://github.com/Safe-DS/Stdlib/blob/92622fb1bac5304c5127bf472e6630aceaf85c16/src/safeds/ml/classical/regression/_elastic_net_regressor.py#L42.

Reply:
Changed the code to only raise an error, when transformers in None and made it raise a warning, when the list is simply empty.